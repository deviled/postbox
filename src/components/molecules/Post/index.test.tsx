import React from 'react';
import { render } from '__tests__/utils';
import { getPostFixture } from '__tests__/mocks/fixtures/posts';
import { Post } from '.';

describe('Post.tsx', () => {
  it('should match snapshot', () => {
    const { message, created_time } = getPostFixture();
    const { baseElement } = render(
      <Post created_time={created_time} message={message} mb={32} />,
    );

    expect(baseElement).toMatchSnapshot();
  });
});
