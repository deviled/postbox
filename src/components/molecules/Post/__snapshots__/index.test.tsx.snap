// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post.tsx should match snapshot 1`] = `
.c1 {
  font-size: 1rem;
  font-weight: 400;
  color: #121212;
  font-weight: bold;
}

.c0 {
  font-size: 1rem;
  padding: 8px 16px;
  border: 1px solid #cccccc;
  margin-bottom: 32px;
}

<body>
  <div>
    <article
      class="c0"
    >
      <p
        class="c1"
        font-weight="bold"
      >
        November 13, 2020 02:02:04
      </p>
      <hr />
      <p>
        initiative bury oak tired bracket habitat save grow desert credit card business product connection master boy respectable sweet depression
      </p>
    </article>
  </div>
</body>
`;
